### PR TITLE
SSJ-11 Verification of Vacancy Committee section while creating an NCI Vacancy Negative Flow

### PR DESCRIPTION
**Pull request by lfcabrera14**
also added Selenium and Playwright tags to SSJ test cases

### CODE REVIEW

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature
Comment (Mariachaudhry) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature:1]: line 21, does it still needed? 
Following line need to format
Line 34, 38, 42, 62, 66, 70, 74

**Review by Mariachaudhry — CHANGES_REQUESTED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Features/OWM_Vacancy_Manager.feature:1]: line 21 is still needed and formatted the rest of the lines

**Review by Mariachaudhry — APPROVED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature [hunk 1]
```diff
@@ -3,33 +3,33 @@ Feature: Unauthenticated applicant Scenarios
   Background: common steps
     Given User is on SSJ home page and user is Unauthenticated Applicant
 
-  @Nekrashevich @SSJ-124  @SSJ-365
+  @Regression @Nekrashevich @SSJ-124  @SSJ-365 @selenium
```
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature:6]: Extra space between @SSJ-124  @SSJ-365

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature [hunk 2]
```diff
@@ -3,33 +3,33 @@ Feature: Unauthenticated applicant Scenarios
...
   Scenario: Not registered option
     And User clicks on Login Dropdown
     And User clicks "Not Registered" option
     Then User is redirected to Okta registration page
 
-  @Nekrashevich @SSJ-31
+  @Regression @Nekrashevich  @SSJ-31 @selenium
```
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature:12]: Extra space between @Nekrashevich  @SSJ-31

**Review by hamidnesar — APPROVED** [review]

**Review by matakevin — CHANGES_REQUESTED** [review]
I think these may have been added either by a prior PR or your current PR but can you get rid of these empty spaces?  Aside from that it looks good

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature (hunk 1 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature]: fixed, thanks!

**Review by matakevin — APPROVED** [review]
LGTM, nice first PR Luis!

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature (hunk 2 above)
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/Unauthenticated Applicant.feature]: done

**Pull request merged**